frontend/src/components/auth/Login.tsx

```diff
@@ -40,59 +41,66 @@ const Login = (): React.ReactElement => {
   }
 
   return (
-    <div style={{ textAlign: "center" }}>
-      <h1>Login</h1>
-      <form>
-        <div>
-          <input
-            type="email"
-            value={email}
-            onChange={(event) => setEmail(event.target.value)}
-            placeholder="username@domain.com"
-          />
-        </div>
-        <div>
-          <input
-            type="password"
-            value={password}
-            onChange={(event) => setPassword(event.target.value)}
-            placeholder="password"
-          />
-        </div>
-        <div>
-          <button
-            className="btn btn-primary"
-            type="button"
-            onClick={onLogInClick}
-          >
-            Log In
-          </button>
-        </div>
-        <GoogleLogin
-          clientId={process.env.REACT_APP_OAUTH_CLIENT_ID || ""}
-          buttonText="Login with Google"
-          onSuccess={(response: GoogleResponse): void => {
-            if ("tokenId" in response) {
-              onGoogleLoginSuccess(response.tokenId);
-            } else {
-              // eslint-disable-next-line no-alert
-              window.alert(response);
-            }
-          }}
-          // eslint-disable-next-line no-alert
-          onFailure={(error) => window.alert(error)}
+    <Grid w="100vw" h="100vh" templateColumns="repeat(2, 1fr)">
+      <GridItem bg="papayawhip">
+        <Image
+          boxSize="50px"
+          m="5vh 5vw"
+          src={CCBCLogo}
+          alt="ccbc-logo"
+          objectFit="cover"
         />
-      </form>
-      <div>
-        <button
-          className="btn btn-primary"
-          type="button"
-          onClick={onSignUpClick}
-        >
-          Sign Up
-        </button>
-      </div>
-    </div>
+        <Image src={LoginGraphic} boxSize="8% 9%" pl="22%" />
+      </GridItem>
+      <GridItem>
+        <Stack justify="center" p="30% 20% 20% 20%">
```

Comment:
are we gonna use [component styling](https://chakra-ui.com/docs/theming/component-style#styling-single-part-components) later?

Reply:
Hmm good point but I'm not sure whether there's anything reusable enough to change into a component like that. Maybe the input fields but there isn't a whole lotta styling added there. @hyunzeekim @chriistine any thoughts on style conventions for stuff like this?

Reply:
Yeah probably the input fields and the button. It would help with maintaining a consistent theme throughout the app, and making modifications would be simpler in that way as well.